README.md

```diff
@@ -45,7 +45,7 @@ pip install .
 ```
 
 ## Requirements
-- Python version 3.7+, **only verified on python 3.7.6**
+- Python version 3.9+, **only verified on python 3.9**
```

Comment:
This readme mention is redundatn; this is where the actual requirement is defined, so we should update that :

https://github.com/sambanova/generative_data_prep/blob/zoltanc/bugfix/update_python_versin_readme/pyproject.toml#L9